docs/developers/smart-contract-tutorial-foundry.md

```diff
@@ -0,0 +1,208 @@
+# Deploying a Smart Contract to Celestia Ethermint with Foundry
+<!-- markdownlint-disable MD013 -->
+
+In this guide you'll learn how to deploy a Solidity smart contract to Celestia Ethermint with [Foundry](https://github.com/foundry-rs/foundry).
+
+## About Foundry
+
+Foundry is a portable, fast and modular toolkit for Ethereum application development.
+
+Foundry is made up of three components:
+
+[__Forge__](https://github.com/foundry-rs/foundry/tree/master/forge): Ethereum testing framework (like Truffle, Hardhat and DappTools).
+
+[__Cast__](https://github.com/foundry-rs/foundry/tree/master/cast): CLI for interacting with EVM smart contracts, sending transactions, and getting chain data.
+
+[__Anvil__](https://github.com/foundry-rs/foundry/tree/master/anvil): Local Ethereum node, similar to Ganache or Hardhat Network.
+
+We'll use all three to create, test, and deploy our Solidity project.
+
+> To learn more about Foundry, check out the [Foundry Book](https://book.getfoundry.sh/).
+
+## Getting started
+
+To deploy to Celestia Ethermint you will need to request tokens from the test faucet.
+
+### Celestia Ethermint Faucet
+
+> USING THIS FAUCET DOES NOT ENTITLE YOU TO ANY AIRDROP OR OTHER DISTRIBUTION OF CELESTIA OR CELESTIA ETHERMINT TOKENS. MAINNET CELESTIA AND CELESTIA ETHERMINT TOKENS DO NOT CURRENTLY EXIST AND THERE ARE NO PUBLIC SALES OR OTHER PUBLIC DISTRIBUTIONS OF ANY MAINNET CELESTIA OR CELESTIA ETHERMINT TOKENS.
```

Comment:
Do we want to use GitHub note syntax? https://github.com/github-community/community/discussions/16925
```suggestion
> **Note**
> USING THIS FAUCET DOES NOT ENTITLE YOU TO ANY AIRDROP OR OTHER DISTRIBUTION OF CELESTIA OR CELESTIA ETHERMINT TOKENS. MAINNET CELESTIA AND CELESTIA ETHERMINT TOKENS DO NOT CURRENTLY EXIST AND THERE ARE NO PUBLIC SALES OR OTHER PUBLIC DISTRIBUTIONS OF ANY MAINNET CELESTIA OR CELESTIA ETHERMINT TOKENS.
```

Reply:
This is how this renders in docusarus
<img width="777" alt="Screen Shot 2022-06-20 at 6 38 31 PM" src="https://user-images.githubusercontent.com/1857282/174689382-197a3e8f-5f81-413d-8ce7-a978946d0817.png">

Reply:
Really? That's odd, I'm pretty sure block quotes are supposed to continue across lines.